Sources/protoc-gen-grpc-swift/main.swift

```diff
@@ -138,16 +147,32 @@ func main(args: [String]) throws {
   for name in request.fileToGenerate {
     if let fileDescriptor = descriptorSet.fileDescriptor(named: name),
        !fileDescriptor.services.isEmpty {
-      let grpcFileName = uniqueOutputFileName(
-        component: "grpc",
-        fileDescriptor: fileDescriptor,
-        fileNamingOption: options.fileNaming,
-        generatedFiles: &generatedFiles
-      )
-      let grpcGenerator = Generator(fileDescriptor, options: options)
-      var grpcFile = Google_Protobuf_Compiler_CodeGeneratorResponse.File()
-      grpcFile.name = grpcFileName
-      grpcFile.content = grpcGenerator.code
+        var binaryFile = Google_Protobuf_Compiler_CodeGeneratorResponse.File()
+        if (options.reflectionService) {
```

Comment:
We're currently only generating reflection data if `!fileDescriptor.services.isEmpty` but we need to generate it for files which only contain messages as well.

Reply:
Should this also be enabled for generating the .pb file?

Reply:
We don't generate `.pb.swift` files, `protoc-gen-swift` does.  _But_ we should generate reflection data for descriptors which only contain messages.